### PR TITLE
refactor(app): switch ODD update modal progress bar with spinner

### PR DESCRIPTION
**Pull request by ncdiehl11**
closes [RQA-2553](https://opentrons.atlassian.net/browse/RQA-2553)

# Overview

switch ODD update modal progress bar with spinner due to inconsistent blue progress fill

# Test Plan

- launch update on ODD
- observe progress modal:
Screenshot 2024-04-09 at 1 47 46 PM

# Changelog

- remove progress bar from update modal and replace with spinner

# Review requests

swappui

# Risk assessment

low

[RQA-2553]: https://opentrons.atlassian.net/browse/RQA-2553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/organisms/UpdateRobotSoftware/UpdateSoftware.tsx [hunk 1]
```diff
@@ -52,6 +48,13 @@ export function UpdateSoftware({
       height="33rem"
       borderRadius={BORDERS.borderRadius12}
     >
+      <Icon
+        name="ot-spinner"
+        size="5rem"
+        spin={true}
```
Comment (koji) [inline app/src/organisms/UpdateRobotSoftware/UpdateSoftware.tsx:54]: Basically, we just pass `spin` to `Icon`

```suggestion
        spin
```

**Review by koji — APPROVED** [review]
lgtm

thank you for fixing!

**Review by mjhuff — CHANGES_REQUESTED** [review]
This needs to target edge. Thanks!

**Review by mjhuff — APPROVED** [review]
Nice, thanks for fixing this!

**Pull request merged**